### ISSUE TITLE
contextMenuBuilder not enable by default,

### ISSUE THREAD

**Issue by geekz-reno** [joanpablo/reactive_forms#398]
paste, copy, select all toolbar not enabled by default on ReactiveTextField (only shown magnifier),

but when config manually like below its working.

```dart
ReactiveTextField(
    formControlName: 'baseApiUrl',
    contextMenuBuilder: (context, editableTextState) {
      return AdaptiveTextSelectionToolbar.buttonItems(
        anchors: editableTextState.contextMenuAnchors,
        buttonItems: <ContextMenuButtonItem>[
          ContextMenuButtonItem(
            onPressed: () {
              editableTextState
                  .copySelection(SelectionChangedCause.toolbar);
            },
            type: ContextMenuButtonType.copy,
          ),
          ContextMenuButtonItem(
            onPressed: () {
              editableTextState
                  .selectAll(SelectionChangedCause.toolbar);
            },
            type: ContextMenuButtonType.selectAll,
          ),
        ],
      );
    },
  )
```

solve using manually triggered contextMenu on every input

```dart
ReactiveTextField(
  formControlName: 'baseStreamUrl',
  keyboardType: TextInputType.url,
  contextMenuBuilder: (context, editableTextState) {
    final items = editableTextState.contextMenuButtonItems;
    return AdaptiveTextSelectionToolbar.buttonItems(
      anchors: editableTextState.contextMenuAnchors,
      buttonItems: items,
    );
  },
)
```

**Comment by byronweiss** [comment]
Another workaround is to sub in the standalone [reactive_text_field](https://pub.dev/packages/reactive_text_field).

**Comment by joanpablo** [comment]
Hi everyone, thanks once again for the support. I have released version 16.1.1 which fixes this issue. Please give it a look and don't hesitate to create a new issue or PR if something else is missing.
